### ISSUE TITLE
Le nœud ne renvoi pas les vacances scolaires en cours.

### ISSUE THREAD

**Issue by replayfactory** [tataille/node-red-french-holidays#16]
Nous sommes dans les vacances de Toussaint (Montpellier / Zone C), le Lundi 21 octobre mais les données renvoyées ne correspondent pas :

```
{
   "day":1,
   "isPublicHoliday":false,
   "isTomorrowPublicHoliday":false,
   "publicHolidayName":null,
   "nextPublicHolidayName":"Toussaint",
   "nextPublicHolidayDate":"01/11/2024",
   "isSchoolHolidays":false,
   "isTomorrowSchoolHolidays":false,
   "schoolHolidaysName":null,
   "schoolHolidaysEndDate":null,
   "nextSchoolHolidaysCoutdownInDays":60,
   "nextSchoolHolidaysName":"Vacances de Noël",
   "nextSchoolHolidaysStartDate":"21/12/2024",
   "nextSchoolHolidaysEndDate":"06/01/2025",
   "schoolPeriod":"2024-2025",
   "year":2024,
   "region":"Métropole",
   "academy":"Montpellier",
   "zones":"Zone C",
   "version":"1.2.3"
}
```

**Comment by 2lastelle** [comment]
Idem Zone A

> object
day: 1
isPublicHoliday: false
isTomorrowPublicHoliday: false
publicHolidayName: null
nextPublicHolidayName: "Toussaint"
nextPublicHolidayDate: "01/11/2024"
isSchoolHolidays: false
isTomorrowSchoolHolidays: false
schoolHolidaysName: null
schoolHolidaysEndDate: null
nextSchoolHolidaysCoutdownInDays: 60
nextSchoolHolidaysName: "Vacances de Noël"
nextSchoolHolidaysStartDate: "21/12/2024"
nextSchoolHolidaysEndDate: "06/01/2025"
schoolPeriod: "2024-2025"
year: 2024
region: "Métropole"
academy: "Grenoble"
zones: "Zone A"
version: "1.2.2"

**Comment by CTPA** [comment]
Bonjour, j'ai le même problème aussi en zone C, Paris

> {"day":1,"isPublicHoliday":false,"isTomorrowPublicHoliday":false,"publicHolidayName":null,"nextPublicHolidayName":"Toussaint","nextPublicHolidayDate":"01/11/2024","isSchoolHolidays":false,"isTomorrowSchoolHolidays":false,"schoolHolidaysName":null,"schoolHolidaysEndDate":null,"nextSchoolHolidaysCoutdownInDays":60,"nextSchoolHolidaysName":"Vacances de Noël","nextSchoolHolidaysStartDate":"21/12/2024","nextSchoolHolidaysEndDate":"06/01/2025","schoolPeriod":"2024-2025","year":2024,"region":"Métropole","academy":"Paris","zones":"Zone C","version":"1.2.3"}

**Comment by tataille** [comment]
Problem identifed. Fix is in progress.

**Comment by fdecourt** [comment]
Je me disais bien que tous mes automatisme basés sur sur noeud étaient lancés alors qu'ils ne devraient pas.
Merci @tataille pour le futur correctif

**Comment by CTPA** [comment]
Oui, ici aussi, j'ai le réveil de mes enfants qui utilise ce nœud, c'est mes filles qui ont fait la tête ce matin :-)
MERCI!
BiG

**Comment by tataille** [comment]
Problem fixed in version 1.2.4. Please update the plugin version is node-red palette manager.

**Comment by tataille** [comment]
Example for Paris (Zone C) with vefrsion 1.2.4

`
{
   "day":1,
   "isPublicHoliday":false,
   "isTomorrowPublicHoliday":false,
   "publicHolidayName":null,
   "nextPublicHolidayName":"Toussaint",
   "nextPublicHolidayDate":"01/11/2024",
   "isSchoolHolidays":true,
   "isTomorrowSchoolHolidays":true,
   "schoolHolidaysName":"Vacances de la Toussaint",
   "schoolHolidaysEndDate":"04/11/2024",
   "nextSchoolHolidaysCoutdownInDays":60,
   "nextSchoolHolidaysName":"Vacances de Noël",
   "nextSchoolHolidaysStartDate":"21/12/2024",
   "nextSchoolHolidaysEndDate":"06/01/2025",
   "schoolPeriod":"2024-2025",
   "year":2024,
   "region":"Métropole",
   "academy":"Paris",
   "zones":"Zone C",
   "version":"1.2.4"
}
`
